### ISSUE TITLE
de novo gene structure prediction

### ISSUE THREAD

**Issue by jwli-code** [yilinZhang-bio/Watermelon-pangenome#1]
Regarding the issue of de novo gene structure prediction, it seems that there is no model for watermelon in the Augustus software. Did you build the model yourself? How can one construct a reasonable model?
`augustus --strand=both --genemodel=complete --singlestrand=false --noInFrameStop=true --protein=on --introns=on --start=on --stop=on --cds=on --codingseq=on --alternatives-from-sampling=true --gff3=on --outfile=${Sample}.gff --uniqueGeneId=true --species=Watermelon genome.fasta`

**Comment by yilinZhang-bio** [comment]
I apologize for the delayed response. I've been traveling and just got the chance to address your question.

You're correct that Augustus doesn't have a built-in model for watermelon. I constructed the model myself. To build a reasonable model for watermelon, I followed the process outlined in the Augustus training tutorial: http://bioinf.uni-greifswald.de/augustus/binaries/tutorial/training.html

This tutorial provides a step-by-step guide on how to create a species-specific model for Augustus. The process involves several key steps:

1. Preparing a set of known gene structures for your species
2. Creating a training set and a test set
3. Training Augustus on your data
4. Testing and optimizing the model

It's a detailed process, but it allows you to create a custom model tailored to the specific characteristics of the watermelon genome.

If you're interested in creating your own model or have any specific questions about the process, I'd be happy to provide more details or clarification.

Again, I apologize for the late reply, and I hope this information is helpful to you.
